frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -25,13 +26,29 @@ export function SDKs({
     subtitle?: string
     stepKey?: OnboardingStepKey
 }): JSX.Element {
-    const { setSourceFilter, setSelectedSDK, setAvailableSDKInstructionsMap, setShowSideBySide, setPanel } =
-        useActions(sdksLogic)
-    const { sourceFilter, sdks, selectedSDK, sourceOptions, showSourceOptionsSelect, showSideBySide, panel } =
-        useValues(sdksLogic)
-    const { productKey } = useValues(onboardingLogic)
+    const {
+        setSourceFilter,
+        setSelectedSDK,
+        setAvailableSDKInstructionsMap,
+        setShowSideBySide,
+        setPanel,
+        setHasSnippetEvents,
+    } = useActions(sdksLogic)
+    const {
+        sourceFilter,
+        sdks,
+        selectedSDK,
+        sourceOptions,
+        showSourceOptionsSelect,
+        showSideBySide,
+        panel,
+        hasSnippetEvents,
+    } = useValues(sdksLogic)
+    const { productKey, product } = useValues(onboardingLogic)
+    const { currentTeam } = useValues(teamLogic)
     const { width } = useWindowSize()
     const minimumSideBySideSize = 768
+    const isFirstProduct = Object.keys(currentTeam?.has_completed_onboarding_for || {}).length === 0
```

Comment:
Would it make more sense to put the `isFirstProductOnboarding` from onboardingStep.tsx into onboardingLogic and reuse?

Reply:
yeah I was just being lazy 😄